docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+services:
+  db:
+    container_name: respa-db
+    build: ./docker/postgres/
+    restart: always
+    environment:
+        POSTGRES_USER: respa
+        POSTGRES_PASSWORD: respa
+        POSTGRES_DB: respa
+    volumes:
+      - db_data:/var/lib/postgresql
+  api:
+    container_name: respa-api
+    build:
+      context: ./
+      dockerfile: ./docker/dev/Dockerfile
+    command: bash -c 'tail -f /dev/null'
+    volumes:
+      - .:/usr/src/app
+    environment:
+      - DEBUG=1
+      - DATABASE_URL=postgis://respa:respa@db/respa
```

Comment:
so, how does it connect to `db` if db container does not have ports exposed? could not make it work this way personally in my project...

Reply:
Good question. Ports seems to be missing indeed, but for some unexplained reason it still works on my machine (perhaps I have something special in my env, which I can't remember any more). Due to it working, probably didn't notice this before.

Anyways, don't have time to debug this atm. Did you try adding the ports? Did it work then?

Reply:
Intra-compose communication happens using Docker's DNS and other network stuff.

Anyway, there's expose in the original Dockerfile: https://github.com/docker-library/postgres/blob/3f585c58df93e93b730c09a13e8904b96fa20c58/9.5/Dockerfile#L172

Reply:
Ah, well that explains. Thanks @jussiarpalahti for linking that 👍. @igordavydsson perhaps you had another problem then.